src/withEditorProps/index.js

```diff
@@ -23,11 +25,57 @@ import { allTypes } from "../utils/annotationTypes";
 
 import { MAX_MATCHES_DISPLAYED } from "../constants/findToolConstants";
 import { defaultMemoize } from "reselect";
+import domtoimage from "dom-to-image";
+// TODO: Fix import
+// -> error Uncaught TypeError: Cannot read property 'apply' of undefined (redux)
+// import { ComponentToPrint } from '../helperComponents/PrintDialog';
 
 // const addFeatureSelector = formValueSelector("AddOrEditFeatureDialog");
 // const addPrimerSelector = formValueSelector("AddOrEditPrimerDialog");
 // const addPartSelector = formValueSelector("AddOrEditPartDialog");
 
+/**
+ * Adds a div in the dom and
+ * renders ComponentToPrint inside
+ *
+ * @return {object} - div element
+ */
+const generateComponentToPrint = () => {
+  const componentToPrint = document
+    .querySelector("div.veVectorInteractionWrapper")
+    .cloneNode(true);
+  // const componentToPrint = <ComponentToPrint editorName="" circular />; // TODO: Pass correct editorName
```

Comment:
Need to uncomment this line. Atm I'm just cloning the active view (circular/linear) in the editor.